### PR TITLE
Denbi brochure

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _posts/2021-02-01-denbi-brochure.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+title: "Two articles related to COVID-19 research by the European Galaxy team in the de.NBI brochure"
+tags: [article, COVID-19]
+supporters:
+ - denbi
+---
+
+<br>
+Two articles by the European Galaxy team have been included in the [de.NBI brochure __Data analysis for the COVID-19 Research__'](https://www.denbi.de/images/Downloads/deNBI_COVID19_brochure.pdf).
```
Comment (bgruening) [inline _posts/2021-02-01-denbi-brochure.md:10]: there is a third one included about Glassgo in Galaxy, do we want to include it here?

Comment (beatrizserrano) [reply _posts/2021-02-01-denbi-brochure.md:10]: as you like, I can change the focus to "Galaxy-related" instead of "written by the EU Galaxy team"

Comment (bgruening) [reply _posts/2021-02-01-denbi-brochure.md:10]: ok, let's do it this way

Comment (beatrizserrano) [reply _posts/2021-02-01-denbi-brochure.md:10]: "this way" means changing the focus?

Comment (bgruening) [reply _posts/2021-02-01-denbi-brochure.md:10]: yes

**Review by bgruening — APPROVED** [review]

**Pull request merged**